### ISSUE TITLE
Update App screens shots in readme

### ISSUE THREAD

**Issue by therajanmaurya** [openMF/self-service-app#2520]
**Summary:**

Summarize your issue in one sentence (what goes wrong, what did you expect to happen).

**Steps to reproduce:**

How can we reproduce the issue?

**Expected behaviour:**

What did you expect the app to do?

**Observed behaviour:**

What did you see instead?  Describe your issue in detail here.

**Device and Android version:**

What make and model device (e.g., Samsung Galaxy S3) did you encounter this on?  What Android
version (e.g., Android 4.0 Ice Cream Sandwich or Android 6.0 Marshmallow) are you running?  Is it
 the stock
version from the manufacturer or a custom ROM?

**Screenshots:**

Can be created by pressing the Volume Down and Power Button at the same time on Android 4.0 and higher.

**Comment by Akshay2004-701** [comment]
I would like to do this task, can I be assigned with it?

**Comment by kmanikanta335** [comment]
@therajanmaurya can i work on this issue ?

**Comment by AvneetSingh2001** [comment]
Hey @Akshay2004-701, as you commented first, so please directly make a PR and send the PR link in the Slack Channel. 
Thanks!

**Comment by Vikashgathala** [comment]
@Akshay2004-701 Can I go with this if you're okay with it?

**Comment by Akshay2004-701** [comment]
ok, sure

**Comment by Vikashgathala** [comment]
Thank you @Akshay2004-701 . @PratyushSingh07 Please assign this to me.

**Comment by PratyushSingh07** [comment]
Go ahead @Vikashgathala

**Comment by Vikashgathala** [comment]
@PratyushSingh07 Do I need to add the screenshots of the same screens and in the same order as previous ones? I am unable to add beneficiaries so same activities with new UI isnt possible right now.

**Comment by PratyushSingh07** [comment]
In that case add UI of those screens that are visible

**Comment by Vikashgathala** [comment]
@PratyushSingh07 Have a look at this. LGTM.

**Comment by PratyushSingh07** [comment]
Instead of loan account details we can show Help section. Moreover, I would prefer having something similar to the one used in Android client. Let me know if you can replicate that

**Comment by Vikashgathala** [comment]
@PratyushSingh07 I'll do this by evening.

**Comment by PratyushSingh07** [comment]
Great, thanks

**Comment by Vikashgathala** [comment]
@PratyushSingh07 Done.

**Comment by PratyushSingh07** [comment]
Lgtm @Vikashgathala . Can you just replace the current help section screenshot with the one in which we can have some of the answers as well. I mean can you expand a few of them . It would look better imo. Let me know if you think otherwise

**Comment by Vikashgathala** [comment]
There is no UI separation between the help section's question and solution after expanding. I tried that but it looks like a single paragraph. This looks a little better and we have any ui improvements in help section in future, it would be better to update it in this.  also, I have squashed the commits and its ready from my side until you find it mandatory to update the screenshots as you are recommending. Let me know what you think. @PratyushSingh07
